libraries/tock-register-interface/README.md

```diff
@@ -262,8 +260,22 @@ ReadWrite<T: IntLike, R: RegisterLongName = ()>
 .matches_all(value: FieldValue<T, R>) -> bool  // Check if all specified parts of a field match
 .extract() -> LocalRegisterCopy<T, R>          // Make local copy of register
 
+Aliased<T: IntLike, R: RegisterLongName = (), W: RegisterLongName = ()>
+.get() -> T                                    // Get the raw register value
+.set(value: T)                                 // Set the raw register value
+.read(field: Field<T, R>) -> T                 // Read the value of the given field
+.read_as_enum<E>(field: Field<T, R>) -> Option<E> // Read value of the given field as a enum member
+.write(value: FieldValue<T, W>)                // Write the value of one or more fields,
+                                               //  overwriting other fields to zero
+.is_set(field: Field<T, R>) -> bool            // Check if one or more bits in a field are set
+.matches_any(value: FieldValue<T, R>) -> bool  // Check if any specified parts of a field match
+.matches_all(value: FieldValue<T, R>) -> bool  // Check if all specified parts of a field match
+.extract() -> LocalRegisterCopy<T, R>          // Make local copy of register
 ```
 
+The `Aliased` type represents cases where read-only and write-only registers,
+with different meanings, are aliased to the same memory location.
+
```

Comment:
I think this is a sufficient explanation of the meaning of `Aliased`. Could it be moved up to the paragraph on lines 229-230? Probably after the first sentence, which is where I as a reader would be wondering what this fourth case meant.

Reply:
Considering `Aliased` usage is an uncommon case, I didn't want to make it especially prominent.

Reply:
I'm not going to block the PR on it or anything, but in my opinion it's worth having the explanation up there at least so that people know it's an uncommon case and something they can ignore.